src/app/core/services/app-user.service.ts

```diff
@@ -0,0 +1,34 @@
+import { HttpClient } from '@angular/common/http';
+import { inject, Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { environment } from '../../../environments/environment';
+
+export interface AppUser {
```

Comment:
Tak jak wcześniej pisałem zmienić na LoggedUser i przenieśc do katalogu models. 
Wgl to nie powinno być w śćeiżce src/app/core/services/app.user.service.ts tylko w src/app/modules/app-users